### DISCUSSION TITLE
Display version of applications in application dashboard - target revision missing

### DISCUSSION THREAD

**Discussion by midestefanis** [argoproj/argo-cd#9020] [Q&A]
How can I see the versions of the microservices in the application dashboard?

**Comment by ishitasequeira** [comment]
Hi @midestefanis, I guess this is a duplicate of https://github.com/argoproj/argo-cd/discussions/9021?
Can we close this one?

**Reply by midestefanis** [reply]
yes, sory, we can close this one
